### PR TITLE
updated cmake system to include lua and new networking example

### PR DESCRIPTION
**Pull request by zaafar**

### CODE REVIEW

File: examples/networking/CMakeLists.txt [hunk 1]
```diff
@@ -6,3 +6,6 @@ install(PROGRAMS ${EXAMPLE_PROGRAMS} DESTINATION share/bcc/examples/networking)
 add_subdirectory(distributed_bridge)
 add_subdirectory(neighbor_sharing)
 add_subdirectory(vlan_learning)
+add_subdirectory(dns_matching)
```
Comment (4ast) [inline examples/networking/CMakeLists.txt]: dns_matching is an example of broken unrolling. pls don't install it.

**Comment by drzaeus77** [comment]
Please squash these into a single commit, it's fine to force push the cleaned git tree to your own tree...the PR will update appropriately, preserving comments.

**Comment by zaafar** [comment]
Sorry for the messy PR. It's clean now.

**Comment by 4ast** [comment]
lgtm

**Pull request merged**